### ISSUE TITLE
RFP Board idea

### ISSUE THREAD

**Issue by vrozansky** [givingforum/givingforum#207]
We discussed this possible task during last week's KM Partners call and there was sufficient interest from participating RAs to open this ticket. We'd like to get a better idea how many RAs are in fact interested in exploring this task further and we'll work out requirements and estimates that will be shared with the group here. At this time, we're only asking for your comments to gauge the level of interest, not official votes. Please respond by March 4th, 2016.

Below is Jon's description of the task to the Advisory Group. It contains more information and a survey that PNW has done on the topic.

Survey: https://www.getfeedback.com/s/obOpph4l 

Highlights:
*	48 respondents
*	Over 70% have open RFP or LOIs
*	25/48 said they'd be likely to post on an RFP board
*	36/48 said they'd be likely to read things on an RFP board

Only well-known existing resource is Foundation Center, which is mostly big/national stuff.
AGM in Boston has already built such a tool, but not shared with the network.

We would definitely would want to partner with nonprofit associations to publicize. (501 Commons has already expressed strong interest). We could easily publish this content in machine-readable form for syndication -- this is a core capability of our website platform.

As an organization that is uniquely situated at the intersection of the nonprofit sector, philanthropic organizations and their communities, Philanthropy Northwest is well positioned to help connect local and regional philanthropic organization's open grantmaking opportunities with a broader nonprofit audience through an "RFP board" feature on our site. 

An RFP board would give the following benefits:
*	A free, tangible benefit to members
*	Helps members learn more about what each other are doing, and thereby can foster collaboration, or at least awareness of opportunities for collaboration and alignment.
*	Aligns with our strategic framework mandate to promote inclusion and transparency in philanthropy
*	If we open it for a fee to non-member posts, it is a minor earned income opportunity. 
*	It is synergistic with our job bank, which already draws a lot of nonprofit eyeballs.
*	It is a good deed by the regional nonprofit community, and represents a low-cost way for philanthropy to be more open and accessible to communities, not unlike our former Northwest Funding Directory, which was too much effort to maintain.

I think this is a low-cost, moderate value effort that leverages our existing technology stack, serves members, serves the field and is both a symbolic and tangible token of our commitment to curating and adding value to information in our network. It is also an interesting pilot test for the bigger idea of a "collaboration opportunity exchange."

**Comment by PhilOhio** [comment]
POH not interested at this time.

**Comment by mtrimbach** [comment]
CNJG is not interested at this time

**Comment by RebekahSeder** [comment]
WRAG might be interested in the future (probably not until 2017), depending on cost.

**Comment by sgeis** [comment]
IPA might be interested in the future. We don't have an immediate need for the RFP board at this time.

**Comment by kristenruff** [comment]
PNY is interested.


______________________________________________
Kristen M. Ruff : Vice President, Member Services
Philanthropy New York
1500 Broadway, 7th Floor, New York, NY 10036
212.714.0699 x 221 | kruff@philanthropynewyork.org<mailto:kruff@philanthropynewyork.org>
www.philanthropynewyork.org<http://www.philanthropynewyork.org/> | @PhilanthropyNY<http://twitter.com/#!/philanthropyny>

Did you know? PHILANTHROPY NEW YORK has MOVED!
Our new home is 1500 Broadway, 7th Floor (at 43rd Street).

**Comment by kcifone** [comment]
CCP feels it is interesting, but not a priority at this time.

**Comment by AClay2** [comment]
PSW might be interested dependent on cost.

**Comment by microveldt** [comment]
PNW is interested dependent on cost and other RAs interested

**Comment by vrozansky** [comment]
Thank you for your feedback. As we discussed on our earlier group call, this shouldn't be a very costly task but we can properly estimate it upon collecting the exact requirements. 

So far we have a handful of RAs that are tentatively interested. Hopefully others will chime in as well. If preference is to postpone this task until later this year, we can certainly do that. Alternatively, we can convene a small working group to work out the requirements, estimate them, and report back to everyone here with the final cost and open it up for voting. I'd gladly facilitate either approach, so please submit your comments and preferences (especially those that have yet to respond). I'd like to hear back from everyone no later than March 18th. Thank you.

**Comment by kristenruff** [comment]
We’re interested in this but have a few other things we need to roll out before turning our attention to this. I’d be in favor of working on this in Q2.

______________________________________________
Kristen M. Ruff : Vice President, Member Services
Philanthropy New York
1500 Broadway, 7th Floor, New York, NY 10036
212.714.0699 x 221 | kruff@philanthropynewyork.org<mailto:kruff@philanthropynewyork.org>
www.philanthropynewyork.org<http://www.philanthropynewyork.org/> | @PhilanthropyNY<http://twitter.com/#!/philanthropyny>

Did you know? PHILANTHROPY NEW YORK has MOVED!
Our new home is 1500 Broadway, 7th Floor (at 43rd Street).

**Comment by EmilyBaron** [comment]
GWP is not interested.
